source/agora/network/NetworkManager.d

```diff
@@ -408,6 +408,13 @@ public class NetworkManager
         }
     }
 
+    /// Returns an already instantiated version of the BanManager
+    /// (please also see `NetworkManager.getBanMananger()`)
+    public BanManager getAlreadyCreatedBanManager () @safe @nogc nothrow pure
```

Comment:
We might want to rename all the `get*` functions to `make*/create*` functions so we would have:

```
makeBanManager() or createBanManager()
getBanManager()
```

We can leave that for a future PR.